app/src/main/res/values-my/strings.xml

```diff
@@ -956,15 +1302,23 @@
     <!-- text for the privacy notice onboarding card header -->
     <string name="onboarding_privacy_notice_header">သင့်အတွက်သီးသန့်လုံခြုံမှု</string>
 
+    <!-- text for the privacy notice onboarding card description
+    The first parameter is the name of the app (e.g. Firefox Preview) Substitute %s for long browser name. -->
+    <string name="onboarding_privacy_notice_description2">သင် အွန်လိုင်းတွင် မျှဝေသည်များ၊ သင် ကျွန်ုပ်တို့ နှင့် မျှဝေသည်များ အတွက် သင်အား စီမံ ခွင့် ပေးနိုင်ရန် ဆောင်ရွက်ထားပါသည်။</string>
```

Comment:
This translation seems to be missing the placeholder.

Reply:
cc @Delphine I am not really sure where the placeholder should be this time, so I cannot suggest a new string. Maybe you can contact the localizer.

Reply:
Doing that now, thanks for flagging

Reply:
Just unblocked the situation until we know where it really has to go... should just be a matter of hours